src/RawTransaction.php

```diff
@@ -980,6 +980,7 @@ public static function validate_signed_transaction($raw_tx, $json_string, $magic
                     foreach ($redeemScript['keys'] as $public_key) {
                         if (self::_check_sig($signature, $message_hash[$i], $public_key) == true) {
                             $pubkey_found = true;
```

Comment:
yea, there's room for improvement but this is enough for now and afaik your focus has shifted to the new lib you've been working on, I'm considering doing that too instead of doing more big overhauls on parts of this lib ;-)